go.mod

```diff
@@ -51,6 +51,11 @@ replace tailscale.com => github.com/coder/tailscale v1.1.1-0.20221117204504-2d65
 // makes importing it directly a bit messy.
 replace github.com/gliderlabs/ssh => github.com/coder/ssh v0.0.0-20220811105153-fcea99919338
 
+// This package is only used in tests, but there's a data race causing
+// occasional flakes.
+// https://github.com/bramvdbogaerde/go-scp/pull/66
+replace github.com/bramvdbogaerde/go-scp => github.com/mafredri/go-scp v1.2.1-0.20221123115109-607fd1f5015c
```

Comment:
This should be forked to `coder` instead of `mafredrei`. Even with no special permissions members of coder can fork/create repos in it.

Reply:
If there's a pull upstream, I don't care if it's on the dev's personal. I just don't want maintained forks on personal ;p.

Reply:
I considered it but don't think it's worth it here since this is a lib only used in tests. Once the change is upstreamed I'll get rid of the repo, if we fork it we'll be stuck on our fork forever. 😅